content/en/docs/features/control-plane/entity-tag.md

```diff
@@ -156,9 +170,19 @@ oc_main_shutdown();
 4. **Absence of Resources after Device Reboot**: ETAGs for non-existing resources are taken into account when calculating the global ETAG during the `oc_etag_load_and_clear` process. This ensures that ETAGs assigned to previous resources are not reused by new resources, thereby avoiding conflicts and maintaining consistency.
 5. **Addition of Resources after Device Reboot**: When new resources are added after a device reboot, the ETAG is set using the algorithm described in the [ETAG Generation Algorithm](#etag-generation-algorithm). This ensures proper tracking of changes for the newly added resource and maintains synchronization with the global ETAG.
 6. **Global ETAG Overflow**: In the event that the global ETAG reaches its maximum value and overflows, all resource ETAGs must be reinitialized. This prevents potential conflicts and ensures the continued accurate tracking of resource changes.
+7. **Software update**: When a software update occurs, the ETAGs are cleared. This prevents potential conflicts and ensures the continued accurate tracking of resource changes.
```

Comment:
What does clearing of ETags mean in this context? Setting them all to 0? Or reseting them all by `oc_etag_get`? Or clearing the storage?

Reply:
fixed -> reseting them all by `oc_etag_get`